Gopkg.toml

```diff
@@ -23,3 +23,7 @@ required = [
 [[constraint]]
   name = "github.com/percona-platform/dbaas-api"
   branch = "pmm-2.16"
+
+[[constraint]]
```

Comment:
it would be good to add comment why we use v6 instead of latest one

Reply:
constraint removed, but actually the version won't change since the latest version of mini-go library uses go modules and dep cannot pull it.